### ISSUE TITLE
Make sure languages "with two parts" are handled correctly

### ISSUE THREAD

**Issue by BaptisteGelez** [Plume-org/rocket_i18n#17]
For instance `en-US` or `pt-BR`

**Comment by fdb-hiroshima** [comment]
I think they should be because of https://github.com/Plume-org/rocket_i18n/blob/master/src/with_rocket.rs#L24 and https://github.com/Plume-org/rocket_i18n/blob/master/src/with_actix.rs#L50 ?

**Comment by BaptisteGelez** [comment]
Yes, but my concern was more about making sure they are working with `gettext-macros` too, and that the two-parts version is correctly used when needed if the one-part version is also available (not sure if I'm clear, sorry).

**Comment by Valodim** [comment]
If I'm reading that code right, the locale part (so `-Lo` out of `ln-Lo`) is simply ignored?

I believe it is very important to support this because of the Chinese language. The locales "Chinese (Traditional)" and "Chinese (Simplified)" are quite different, but share the `zh` language prefix.
